src/components/HeroBanner/HeroBanner.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { forwardRef, FC } from 'react';
```

Comment:
FC is no longer needed 🙂 

```suggestion
import React, { forwardRef } from 'react';
```

Reply:
interested to know why that didnt throw a linting error :(

Reply:
hmmmm `@typescript-eslint/no-unused-vars` from `"plugin:@typescript-eslint/recommended"` was throwing a warning, overriding our rules of an error. Might have to rework the eslint / typescript config / ordering

Reply:
It did but as a warning
https://github.com/aesop/aesop-gel/pull/374/checks?check_run_id=2224331551#step:5:31